config/locales/views/en.yml

```diff
@@ -454,7 +454,24 @@ en:
         twitter: "@AO3_Status Twitter feed"
   home:
     donate:
+      general:
+        text: There are two main ways to support the AO3 - donating your time or money.
+        title: Donations
+      money:
+        donation_url: https://donate.transformativeworks.org/otwgive
+        donation_url_text: donation to the OTW
+        text_html: The AO3 has ongoing running costs - electricity for the servers and bandwidth so you can reach them - and one-off costs such as buying new servers as the number of users and works increases. Any %{donation_url} is a big help. (Don't worry, we'll never connect your AO3 username and your financial information.)
+        title: Donating Financially
       page_title: Donate or Volunteer
+      time:
+        otw_url: http://www.transformativeworks.org
+        otw_url_text: Organization for Transformative Works
+        projects_url: http://www.transformativeworks.org/our-projects
+        projects_url_text: our projects
+        text_html: The %{otw_url} is the parent organization of the Archive of Our Own. We are often looking for volunteers to contribute to %{projects_url}. If you're interested in volunteering specifically to help the Archive of Our Own, the roles to look out for include tag wranglers, testers, coders, Support staff and Abuse staff. We encourage you to browse through the %{volunteer_url} and apply for any roles which match your qualifications and interests.
+        title: Donating Your Time
+        volunteer_url: http://www.transformativeworks.org/volunteer
+        volunteer_url_text: available listings
```

Comment:
While moving the url themselves to their own key is likely good, I feel like, in the context of an external URL, extracting the link text might be a bit excessive, ending with small sentence fragments puzzling on their own, compared to something more straightforward like:
```rb
t(".money.text_html", donation_url: t('.money.donation_url'))
```
```yml
money:
    donation_url: https://donate.transformativeworks.org/otwgive
    text_html: The AO3 has ongoing running costs - electricity for the servers and bandwidth so you can reach them - and one-off costs such as buying new servers as the number of users and works increases. Any <a href="%{donation_url}">donation to the OTW</a> is a big help. (Don't worry, we'll never connect your AO3 username and your financial information.)
```